Resources/Prototypes/Entities/Markers/Spawners/Random/Decals/flora.yml

```
@@ -13,6 +13,7 @@
     - FloorJungleAstroGrass
     - FloorAstroIce
     - FloorAstroSnow
+    - FloorMowedAstroSnow # Delta-V
```

nit: Just label as `# DeltaV`


```suggestion
    - FloorMowedAstroSnow # DeltaV
```